### ISSUE TITLE
export route as fit file 

### ISSUE THREAD

**Issue by Whoptimus** [osmandapp/Osmand#6064]
A lot of bicycle computer manufactures are supporting fit route files (Garmin, Wahoo, Stages, Pioneer, Bryton, etc).  

It would be nice if osmand will export routes in fit file format, so that we can use osmand like a kind of a mobile basecamp. 

The gpx who Osmand generates when saving a route, allready haves the information a Fit route file needs.

**Comment by golserma** [comment]
Yes please I'd like to share a calculated route brouter to my cycling computer.

**Comment by Fmstrat** [comment]
Some context here: There are literally thousands of complaints from Garmin users that when they import a GPX with waypoints to their watch or bike computer, Garmin kills the waypoints. However, Garmin does not do that for FIT files.

Supporting the FIT format for export would open up OSMAnd to a huge swath of people looking for a way to do this without paying a monthly subscription to RideWithGPS or similar services.

Long time user, and would absolutely love to see this added.

**Comment by nodje** [comment]
Can't wait for this 👍
